torch/jit/__init__.py

```diff
@@ -2035,6 +2035,10 @@ def register_all(mod):
     if PY37:
         _builtin_table[id(math.remainder)] = "aten::mathremainder"
 
+    import torch.distributed.autograd as dist_autograd
```

Comment:
This might be better organized as an if statement that appends to the _builtin_ops table, rather than another special case here. In face, all the manually uses of _builtin_table here really should be manual modifications to _buildin_ops.